Gemfile.lock

```diff
@@ -0,0 +1,261 @@
+GEM
```

Comment:
This is automatically generated when you run ``bundle exec jekyll serve`` so probably not absolutely necessary, but leaving it regardless.